news/2018-06-12-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,353 @@
+
+*written by [hikikochan](https://osu.ppy.sh/u/6512678)*
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/kata.jpg)](https://osu.ppy.sh/u/6651672)
+
+It is a bit surprising to see **[katacheh]( https://osu.ppy.sh/u/6651672)** here as he only ranked one map in 2017 and the least we can say is that this map probably did not really influence other mappers. On the other hand, **[katacheh]( https://osu.ppy.sh/u/6651672)** is definitely one of the most impressive gimmick mappers if not the best of 2017 as he completely revolutionized the way of using sliders and SV changes in taiko mapping. If you wish to experience some of his crazy creations I suggest that you check some of his loved maps such as **[Cirno's Perfect Math Class – Ryu* remix -](https://osu.ppy.sh/s/481441)** and **[Artificial Intelligence : Alpha ~ *GO!!]( https://osu.ppy.sh/s/537827)**.
+
+*written by [Nofool](https://osu.ppy.sh/u/672430)*
+
+<a name="catch" id="catch">![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/mode_catch.jpg)</a>
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/on-the-fm.jpg)](https://osu.ppy.sh/s/609679)
+
+Among the 100 mapsets, **[On the FM](https://osu.ppy.sh/s/609679)** by **[Ascendance](https://osu.ppy.sh/u/2931883)** and other contributing mappers stood firmly on the podium and is greeted with the Grand Map Award. Featuring a detailed map spread of limit-breaking difficulties with energetic hitsounding style, On the FM proved its ability to bring back the beat and get everyone shaking.
+
+*written by [alienflybot](https://osu.ppy.sh/u/636114)*
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/cycle-hit.jpg)](https://osu.ppy.sh/s/692367)
+
+**[Spectator](https://osu.ppy.sh/u/702598)** and **[Ascendance](https://osu.ppy.sh/u/2931883)** hit a home run with their award-winning marathon, **[Cycle Hit](https://osu.ppy.sh/s/692367)**. This map offers a wide array of patterns with aesthetic designs while maintaining a consistent map structure. An upbeat and compact atmosphere is brought up during the gameplay. This is something that you don't want to miss out.
+
+*written by [alienflybot](https://osu.ppy.sh/u/636114)*
+
+[![](/wiki/shared/news/2018-06-12-mappers-choice-awards-2017-results/7-colors.jpg)](https://osu.ppy.sh/s/612303)
+
+The outstanding mapset **[7 Colors](https://osu.ppy.sh/s/612303)** definitely added some colors to the palette of mapping. Pushed to the ranked status single-handedly by **[Sorcerer](https://osu.ppy.sh/u/1913190)**, the mapset quality remained exceptional. You may experience precise usage of patterns with impressive flows through the gameplays. The best work from Sorcerer is yet to come.
```

Comment:
-> the mapset quality is exceptional with precise usage of patterns and impressive flow.